GAE/src/org/akvo/flow/rest/security/EntryPoint.java

```diff
@@ -25,26 +25,24 @@
 import org.springframework.security.core.AuthenticationException;
 import org.springframework.security.web.AuthenticationEntryPoint;
 
-import com.gallatinsystems.common.util.PropertyUtil;
+import com.google.appengine.api.utils.SystemProperty;
 
 public class EntryPoint implements AuthenticationEntryPoint {
-    private final AuthenticationEntryPoint defaultEntryPoint;
-    private final AuthenticationEntryPoint alternativeEntryPoint;
-    private final String defaultAuthProvider;
-
-    public EntryPoint(AuthenticationEntryPoint defaultEntryPoint, AuthenticationEntryPoint alternativeEntryPoint) {
-        this.defaultEntryPoint = defaultEntryPoint;
-        this.alternativeEntryPoint = alternativeEntryPoint;
-        this.defaultAuthProvider =PropertyUtil.getProperty("defaultAuthProvider");
+    private final AuthenticationEntryPoint devEntryPoint;
+    private final AuthenticationEntryPoint prodEntryPoint;
+
+
+    public EntryPoint(AuthenticationEntryPoint prodEntryPoint, AuthenticationEntryPoint devEntryPoint) {
+        this.devEntryPoint = devEntryPoint;
+        this.prodEntryPoint = prodEntryPoint;
     }
 
     @Override
     public void commence(HttpServletRequest request, HttpServletResponse response, AuthenticationException authException) throws IOException, ServletException {
-        if ("auth0".equals(defaultAuthProvider)) {
-            alternativeEntryPoint.commence(request, response, authException);
+        if (SystemProperty.environment.value() == SystemProperty.Environment.Value.Production) {
```

Comment:
Explicit configuration please, so that developers can chose which auth method to use locally.